ansible/roles/ansible-tvault-contego-extension/tasks/uninstall_rhel.yml

```diff
@@ -7,9 +7,17 @@
   service: name=tvault-contego state=stopped enabled=no
   ignore_errors: yes
 
-- name: uninstall tvault contego extension packages
+- name: uninstall tvault contego extension packages when using python2
   yum:
      name:
         - tvault-contego
         - puppet-triliovault
```

Comment:
puppet-triliovault should get installed first than tvault-contego.

Reply:
requested changes done